task-maker-format/src/ioi/statement/templates/booklet.tex

```diff
@@ -13,6 +13,7 @@
 %{packages}%
 
 \begin{document}
+  %{intro_page}%
```

Comment:
This file seems to be indented with tabs... I know 🙈

Reply:
fixed